mmv1/products/securesourcemanager/Instance.yaml

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Instance'
+kind: 'securesourcemanager#instance'
+base_url: 'projects/{{project}}/locations/{{location}}/instances?instance_id={{instance_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/instances/{{instance_id}}'
+update_verb: :PATCH
+update_mask: true
+description: 'Instances are deployed to an available Google Cloud region and are accessible via their web interface.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/secure-source-manager/'
```

Comment:
The documentation should be `https://cloud.google.com/secure-source-manager/docs/create-instance`

Reply:
After talked offline, the link `https://cloud.google.com/secure-source-manager/overview/overview` can be used here.